src/js/game/LevelMVC/LevelView.js

```diff
@@ -1111,8 +1165,16 @@ module.exports = class LevelView {
     }
   }
 
+  // Ice and glass blocks do not drop mini-blocks in Minecraft.
+  shouldDropMiniBlock(blockType) {
+    return !blockType.startsWith("ice") && !blockType.startsWith("glass");
```

Comment:
Can we add test coverage for these two cases as well?

Reply:
Sure thing!

Reply:
Added unit tests for both of these and they're passing, so PTAL? This should be clear to go I think.

Reply:
This seems pretty redundant. Given that `this.createMiniBlock` will simply return `null` for these values, we can just use `if (sprite)` to determine if we should be dropping a mini block.

I'll also point out that the return from `this.createMiniBlock` is determined by the definition of `LevelBlock.getMiniblockFrame`, which is already doing the work of specifying that ice and glass do not drop mini blocks. Redefining that here is definitely not great

Reply:
Removed and added the check in createMiniBlock()